examples/sha1/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "hacspec-sha1"
+version = "0.1.0"
+authors = ["PLR <51248199+plredmond@users.noreply.github.com>"]
```

Comment:
Is this the email address you want to use?

Reply:
Yes